ui/StatusQ/src/StatusQ/Components/StatusChatList.qml

```diff
@@ -4,7 +4,7 @@ import QtQuick.Controls 2.15 as QC
 
 import StatusQ.Core 0.1
 import StatusQ.Core.Theme 0.1
-import StatusQ.Core.Utils 0.1
+import StatusQ.Core.Utils 0.1 as SQUtils
```

Comment:
You don't need to do it inside StatusQ itself, you can't import the app's `utils` from here